package.json

```diff
@@ -22,6 +22,7 @@
     "gh-build-nightly": "webpack --config webpack.dev.js --env env=production basename=/website-3d-cell-viewer",
     "gh-build-release": "webpack --config webpack.dev.js --env env=production basename=/website-3d-cell-viewer-release",
     "s3-build": "webpack --config webpack.dev.js --env env=production",
+    "docker-build": "webpack --config webpack.dev.js --env env=production",
```

Comment:
Same instruction as `s3-build` but it seemed unsafe to reuse it directly.

Reply:
still, any changes to webpack.dev.js will potentially cause issues because shared across multiple build types.  Might be a good thing to note in one of our readme or CONTRIBUTING type of docs.

Reply:
Hm, now that I'm thinking about it most of these build commands likely don't need to use the dev webpack config. I'll test removing it for docker and open an issue for this.

Reply:
See #247!